private/model/api/load.go

```diff
@@ -221,6 +221,7 @@ func (a *API) Setup() error {
 
 	a.findEndpointDiscoveryOp()
 	a.injectUnboundedOutputStreaming()
+	a.enableGeneratedTypedErrors()
```

Comment:
Nit: A comment can be added here where to highlight the supported protocols i.e. Json and rest-json.